pkg/trebuchet/server/rest/reststorage.go

```diff
@@ -0,0 +1,72 @@
+package rest
+
+import (
+	"context"
+
+	trebuchet_v1 "github.com/atlassian/voyager/pkg/apis/trebuchet/v1"
+	"go.uber.org/zap"
+	metainternalversion "k8s.io/apimachinery/pkg/apis/meta/internalversion"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/registry/rest"
+)
+
+type REST struct {
```

Comment:
This struct is not used anywhere. See how it's injected in creator: https://github.com/atlassian/voyager/blob/master/pkg/creator/server/apiserver/apiserver.go#L92-L95